src/cljs/org/broadinstitute/firecloud_ui/page/billing/create_project.cljs

```diff
@@ -18,20 +17,14 @@
       {:header "Create Billing Project"
        :content
        (react/create-element
-        (let [{:keys [billing-accounts billing-acct-error]} @state]
+        (let [{:keys [billing-accounts billing-acct-error get-error-details]} @state]
           (cond billing-acct-error
-                ; if the user has not enabled the correct scopes to list billing accounts, the call will return an error
-                ; with a redirect URL in string-encoded JSON format.  Rather than redirecting the browser directly
-                ; (not permitted in AJAX) we pop up a separate window with a callback to this component's :get-billing-accounts
-                (try
-                  (if-let [redirect-url ((utils/parse-json-string (billing-acct-error "message")) "redirect")]
-                    [comps/Button {:text "Click Here To Enable Billing Permissions"
-                                   :onClick #(.. js/window
-                                                 (open redirect-url
-                                                       "Authentication"
-                                                       "menubar=no,toolbar=no,width=500,height=500"))}]
-                    [comps/ErrorViewer {:error billing-acct-error}])
-                  (catch js/Object _ [comps/ErrorViewer {:error billing-acct-error}]))
+                (if (= billing-acct-error 403)
+                  [:div {:style {:textAlign "center"}}
+                   "Billing permissions are not enabled." [:br] [:br]
+                   [comps/Button {:text "Enable Billing Permissions"
+                                  :onClick #(react/call :.enable-billing-permissions this)}]]
+                  [comps/ErrorViewer {:error (get-error-details)}])
```

Comment:
I believe this is a request for **all** billing accounts, not a specific one. As far as I know, I didn't change the behavior of this, only the mechanics.